documentation/src/main/asciidoc/topics/proc_using_spring_cache_provider.adoc

```diff
@@ -1,10 +1,8 @@
-[id='adding-spring-cache_{context}']
-= Using {brandname} as a Spring cache provider
+[id='using-spring-cache-provider_{context}']
+= Using {brandname} as a Spring Cache provider
 
-To add caching support to your application, use the following annotations:
-
-*  adds entries to the cache.
-* `@CacheEvict` removes entries from the cache.
+After adding {brandname} dependencies, you can enable cache abstraction of Spring by adding the `@EnableCaching` annotation to your configuration classes.
+This allows you to start using the `@Cacheable` on the methods that you want to retrieve cached values from.
```

Comment:
Another nit on the abstract, I'd keep it short and sweet for the minimalism win.

Add the `@EnableCaching` annotation to one of your configuration classes and then add the `@Cacheable` and `@CacheEvict` annotations to use remote or embedded caches.